### ISSUE TITLE
rz-test: REGEXP_FILTER_ERR should match all lines

### ISSUE THREAD

**Issue by XVilka** [rizinorg/rizin#4088]
Currently, if we have output like these strings:
```
 WARNING: Function name 'func' already exists
 ERROR: Cannot add function (duplicated)
 WARNING: Function name 'func' already exists
 ERROR: Cannot add function (duplicated)
```
and filter like this: `REGEXP_FILTER_ERR=ERROR.*`, we will only match one line and not the rest, thus allowing this output:
```
 ERROR: Cannot add function (duplicated)
 WARNING: Function name 'func' already exists
 ERROR: Cannot add function (duplicated)
```

**Comment by deadshot8086** [comment]
same is with `REGEXP_FILTER_OUT=ERROR.*`

**Comment by deadshot8086** [comment]
related to outdated regex util, already addressed in #4033

**Comment by XVilka** [comment]
Fixed by https://github.com/rizinorg/rizin/commit/5afc51f0e2f172b0d4ba14d28ed415c60b5c8761
